apollo-openapi/src/main/java/com/ctrip/framework/apollo/openapi/api/AppOpenApiService.java

```diff
@@ -18,13 +18,22 @@
 
 import com.ctrip.framework.apollo.openapi.dto.OpenAppDTO;
 import com.ctrip.framework.apollo.openapi.dto.OpenEnvClusterDTO;
+import java.util.Collections;
 import java.util.List;
 
 /**
  * @author wxq
  */
 public interface AppOpenApiService {
 
+  default void createApp(OpenAppDTO openAppDTO) {
+    throw new UnsupportedOperationException();
+  }
+
+  default void createApp(String env, OpenAppDTO openAppDTO) {
```

Comment:
The app's scope is global. Could you share the situation where this method applies?

Reply:
For example, there are env: `SIT`, `UAT`

User create app1 already in those env.

When user add a new env `DEV`, if user want to create app1 in `DEV` too, will this method applies?

Reply:
Adding a new environment is somewhat complex as app/appnamespace/cluster/namespace should also be taken into consideration. Please refer to [2.1.2.4 Importing ApolloConfigDB project data from another environment](https://www.apolloconfig.com/#/en/deployment/distributed-deployment-guide?id=_2124-importing-apolloconfigdb-project-data-from-another-environment). Therefore, I believe we could exclude this from the open API.